UDPClientGUI/MainWindow.xaml.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+
+namespace UDPCommGUI
+{
+    /// <summary>
+    /// Interaction logic for MainWindow.xaml
+    /// </summary>
+    public partial class MainWindow : Window
+    {
+
+        MyUDPClient client = new MyUDPClient();
```

Comment:
Use `var` where possible. In C#, if you have the type on the right hand side you usually don't need the type on the left hand side.

There are a few cases where you'd explicitly define the type, here are 2, but there may be more.
- When it's not initialised at the time of declaration
- For `Actions/Func` - check these out

Reply:
Changed as suggested.

Reply:
Ignore that, my mistake - can't use `var` with class fields. You can resolve this conversation.